### ISSUE TITLE
addField with an initial value no longer working

### ISSUE THREAD

**Issue by stevengoldberg** [erikras/redux-form#630]
Calling `addField` and passing an object for the initial value doesn't seem to be working for me, although it was working in a recent version.

My form has fields like this: 

```
[
    'acknowledgments.items[].number',
    'acknowledgments.items[].name',
    'acknowledgments.show',
]
```

When I call `fields.acknowledgments.items.addField({ number: 1, name: 'foo' })`, the `redux-form/ADD_ARRAY_VALUE` action shows a `path` of `acknowledgments.items` and a `value` of the object I passed, but it shows an empty array for `fields`, which seems to be the problem.

**Comment by flaub** [comment]
PR incoming for a fix (I probably broke this with a recent fix).

**Comment by erikras** [comment]
@flaub's fix published as [`v4.1.9`](https://github.com/erikras/redux-form/releases/tag/v4.1.9). Thanks to you both. :+1:
